File: python/easy/fizz_buzz.py
def fizz_buzz(n):
  for item in range(1, n + 1):
    if (item % 3 == 0 and item % 5 == 0):
      print('FizzBuzz')
    elif (item % 5 == 0):
      print('Buzz')
    elif (item % 3 == 0):
      print('Fizz')
fizz_buzz(25)

File: dart/easy/fizzBuzz.dart
class FizzBuzz {
  int count;

  FizzBuzz({required this.count});

  int get fizzBuzzCount {
    return count;
  }

  List<String> fb() {
    List<String> arr = [];
    for (int i = 1; i <= count; i++) {
      if (i % 3 == 0 && i % 5 == 0) {
        arr.add('FizzBuzz');
      } else if (i % 5 == 0) {
        arr.add('Buzz');
      } else if (i % 3 == 0) {
        arr.add('Fizz');
      } else {
        arr.add(i.toString());
      }
    }
    return arr;
  }
}
